docker-hub/members.md

```diff
@@ -18,11 +18,12 @@ Use the following steps to invite members to your organization via Docker ID or
 1. Go to **Organizations** in [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}, and select your organization.
 2. In the **Members** tab, select **Invite Member**.
 3. Select **Emails Or Docker IDs**.
-4. Enter the invitee’s Docker ID or email address to invite, and select a team from the drop-down list.
+4. Enter up to 1000 Docker IDs or email addresses to invite. Separate multiple entries using a space or comma.
```

Comment:
Can we say: Enter the applicable Docker IDs or email addresses you want to invite, up to the max of 1000. Separate the multiple entries using a space or comma.

Reply:
Thank you. Updated in the feedback commit below.